From f290218a6fd4e2b849875f6d465eaa861ec02aff Mon Sep 17 00:00:00 2001
From: Martin Liska <mliska@suse.cz>
Date: Fri, 25 Mar 2022 14:40:37 +0100
Subject: [PATCH] Include <iterator>.

In order to fix:
[   16s] /home/abuild/rpmbuild/BUILD/libkleo-21.12.3/src/kleo/keyserverconfig.cpp:73:44:   required from here
[   16s] /home/abuild/rpmbuild/BUILD/libkleo-21.12.3/src/utils/algorithm.h:35:30: error: no matching function for call to 'begin(QStringList&)'
[   16s]    35 |     std::transform(std::begin(c), std::end(c), std::begin(c), op);
[   16s]       |                    ~~~~~~~~~~^~~

The change is documented here:
https://gcc.gnu.org/gcc-12/porting_to.html#header-dep-changes
---
 src/utils/algorithm.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/utils/algorithm.h b/src/utils/algorithm.h
index 6fb8669..1bcab2c 100644
--- a/src/utils/algorithm.h
+++ b/src/utils/algorithm.h
@@ -11,6 +11,7 @@
 #pragma once
 
 #include <algorithm>
+#include <iterator>
 
 namespace Kleo
 {
-- 
GitLab

